patient-view-be/src/main/java/spvproject/patientviewbe/controller/AnamnesisController.java

```diff
@@ -0,0 +1,26 @@
+package spvproject.patientviewbe.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import spvproject.patientviewbe.model.Anamnesis;
+import spvproject.patientviewbe.repository.AnamnesisRepository;
+import java.util.List;
+
+@RestController
+@RequestMapping("/anamnesis")
+public class AnamnesisController {
+
+    @Autowired
+    AnamnesisRepository anamnesisRepository;
+
+    @GetMapping("/get-all")
```

Comment:
This can be simply named `/all`. Then the request would be `GET /anamnesis/all`. At the moment this is a pleonasm as it is `GET /anamnesis/get-all` :P you get 2 times